package.json

```diff
@@ -1,12 +1,10 @@
 {
   "name": "github-action-for-generator",
-  "version": "2.1.25",
-  "description": "GitHub action to generate all the things form your AsyncAPI document",
-  "main": "lib/index.js",
+  "version": "3.0.0",
+  "description": "GitHub action to perform various actions on AsyncAPI document using CLI",
+  "main": "entrypoint.sh",
   "scripts": {
-    "lint": "eslint --max-warnings 0 --config .eslintrc lib",
-    "test": "jest --coverage",
-    "start": "node lib/index.js",
+    "test": "make test",
     "generate:assets": "echo 'No additional assets need to be generated at the moment'",
     "bump:version": "npm --no-git-tag-version --allow-same-version version $VERSION",
```

Comment:
I guess we do not need it, especially that versioning of this action will not me recorded in this file - it is purely development related file. You can also remove `version` property and in `description` specify purpose if that file

you can also remove `package-lock.json` as there are no dependencies to track here

Reply:
I guess node version bump can also be removed from actions then.